mkversion.sh

```diff
@@ -108,6 +111,10 @@ fi
 # append dirty at the end if we had a dirty tree
 if [ "$DIRTY" = "true" ]; then
     v="$v-dirty"
+    (
+      git status
+      git diff
+     ) > "$PKG_BUILDDIR"/data/dirty-git-tree-info.txt
```

Comment:
To answer your question, if the tree is dirty, my opinion is we should always show it. The notion that the tree is dirty is no help in itself, and unless I am ignorant (I am), then as the action will always fail, then there seems to be no case where we expect a dirty tree.